### PR TITLE
Enable copying comm code

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

https://transformuk.atlassian.net/browse/HOTT-671

### What?

I have added/removed/altered:

- [x] Enable easy copying of a commodity code

### Why?

I am doing this because:

- This is required to make the service slightly easier to use

### Heading that acts like a commodity (https://www.trade-tariff.service.gov.uk/headings/9703)

Screenshot from 2021-05-18 11-13-51

### Heading that does not act like a commodity

Screenshot from 2021-05-18 11-13-38

### Commodity

Screenshot from 2021-05-18 11-14-29

### CODE REVIEW

File: app/webpacker/src/stylesheets/_copy_code.scss [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+.tariff-breadcrumbs nav ul li div.copy_code span {
+    padding-right: 0px;
+}
+
+.copy_code a {
+    width: 171px;
+}
+
+.tariff-breadcrumbs nav ul li {
```
Comment (krios82) [inline app/webpacker/src/stylesheets/_copy_code.scss]: Is this needed?

File: app/webpacker/src/stylesheets/_copy_code.scss [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    text-indent: -999em;
+    margin-top  : 5px;
+    padding-left: 1.75em;
+}
+
+.tariff-breadcrumbs nav ul li div.copy_code span {
+    padding-right: 0px;
+}
+
+.copy_code a {
```
Comment (krios82) [inline app/webpacker/src/stylesheets/_copy_code.scss]: This is also probably a leftover, there's no `a` child in the copy code

File: app/webpacker/src/stylesheets/_copy_code.scss [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+.copied {
+    display     : block;
+    text-indent: -999em;
+    margin-top  : 5px;
+    padding-left: 1.75em;
+}
+
+.tariff-breadcrumbs nav ul li div.copy_code span {
```
Comment (krios82) [inline app/webpacker/src/stylesheets/_copy_code.scss]: Is this needed?

**Review by krios82 — APPROVED** [review]
There are a few css bits that don't seem relevant, but apart from that looks good to me

File: app/views/shared/_search.html.erb [hunk 4]
```diff
@@ -2,7 +2,15 @@
 
 <% if (controller_name == 'commodities' && action_name == 'show') || (controller_name == 'headings' && action_name == 'show' && @heading&.declarable?) %>
   <header>
-    <h1 class="govuk-heading-l commodity-header">Commodity information for <%= (@commodity || @heading).code %></h1>
+    <% comm_code = (@commodity || @heading).code %>
```
Comment (tavyy) [inline app/views/shared/_search.html.erb]: Can we move this to a helper instead?

**Review by tavyy — APPROVED** [review]

**Pull request merged**